capture.cpp

```diff
@@ -1657,7 +1703,10 @@ const char *locale = DEFAULT_LOCALE;
     printf(" Darkframe: %s\n", yesNo(darkframe));
     printf(" Debug Level: %d\n", debugLevel);
     printf(" TTY: %s\n", yesNo(tty));
-    printf("%s\n", KNRM);
+    printf(" Version 0.8 Exposure Method: %s\n", yesNo(use_new_exposure_algorithm));
+
+    // This should always be last since it turns color off.

```

Comment:
Perhaps break up that functionality? Print the SDK version with one printf(), then turn off the color with a second, so it's clear that the latter is for restoring the terminal to normal.